myblog/forms.py

```diff
@@ -0,0 +1,7 @@
+from django import forms
+from . import models
```

Comment:
O que acha de usar imports absolutos?
Na maioria dos casos aqui no Automation usamos esse tipo de import justamente para deixar explicito.
```suggestion
from models import Post
```